src/style.css

```diff
@@ -1 +1,40 @@
 /* add styles here */
+
+.page__body {
+  margin: 0;
+  width: 100px;
+  height: 116px;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
There's no need to use `display: flex;` on `.stars` if you're not applying any flex properties to it. If you intended to use flexbox, consider applying properties like `justify-content`, `align-items`, or `flex-direction`.